test/helpers/fixture_sets/hello_cloud.rb

```diff
@@ -10,6 +10,7 @@ def assert_all_up
       assert_unmanaged_pod_statuses("Succeeded")
       assert_all_web_resources_up
       assert_all_redis_resources_up
+      assert_all_memcached_resources_up
```

Comment:
This doesn't actually test your code added at all, since the fixtures you added contain a regular deployment/service (which already have plenty of coverage). Our CI is not running Shopify's closed-source controllers, so there sadly isn't a way to add tests for TPR/CRD support. As a result, changes to TPR/CRD support need to be manually tophatted against tierstaging; if you need pointers for that, please ping me on slack. Our long-term solution for this (which we'll be able to get started on very soon) is to make the CRD support generic: https://github.com/Shopify/kubernetes-deploy/issues/128.

Reply:
Okay good to know 👍